### DISCUSSION TITLE
Confusion over `display_duration`

### DISCUSSION THREAD

**Discussion by dbatten5** [stevearc/overseer.nvim#354] [Q&A]
In my head, enabling this option would display a running duration of the task somewhere but this doesn't seem to be the case. Am I misunderstanding this? My config with toy example looks like this:
```
return {
  "stevearc/overseer.nvim",
  lazy = false,
  keys = {
    {
      "<leader>O",
      ":OverseerRun<cr>",
      desc = "Open the Overseer menu",
      silent = true,
    },
  },
  config = function()
    local overseer = require("overseer")

    overseer.register_template({
      name = "Sleep",
      builder = function()
        local cmd = { "sleep", "3" }
        return {
          cmd = cmd,
          components = {
            "default",
            {
              "on_output_quickfix",
              open_on_exit = "failure",
            },
          },
        }
      end,
      condition = {
        filetype = "python",
      },
    })

    overseer.setup({
      templates = {},
    })
  end,
}
```

**Comment by stevearc** [comment]
The default components attached to a task can be found here:
https://github.com/stevearc/overseer.nvim/blob/44e927ec68d3cfb012e2e9dba3dc79a3d68fadde/lua/overseer/config.lua#L148
You will notice that the configuration for `display_duration` sets the detail level at `2`. This means that the duration won't be visible until you increase the detail level in the task list at least once (press `?` in the task list to see all the keymaps).

You can customize this by changing the definition of the `default` component alias in your own config and pass a `detail_level = 1` (or exclude it, because the default is 1).
